client/blocks/upload-image/constants.js

```diff
@@ -0,0 +1,7 @@
+export const ALLOWED_FILE_EXTENSIONS = [
```

Comment:
Good catch, I'll copy that list (since it also contains other media types not supported here).